CHANGELOG.md

```diff
@@ -77,30 +78,30 @@ Notes on dependencies:
 
 New methods:
 
-- Added `count_geometries` method from shapely to GeoSeries/GeoDataframe (#3154).
```

Comment:
I've noticed incorrect capitalisation so I fixed that here rather than making a separate PR. Can split it if needed.